web_widget_color/controller.py

```diff
@@ -0,0 +1,61 @@
+# -*- encoding: utf-8 -*-
+############################################################################
+#
+# OpenERP, Open Source Web Color
+# Copyright (C) 2012 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# Copyright (C) 2014 Anybox <http://anybox.fr>
+# Copyright (C) 2015 Taktik SA <http://taktik.be>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# @author Étienne Beaudry Auger <etienne.b.auger@savoirfairelinux.com>
+# @author Adil Houmadi <ah@taktik.be>
+#
+##############################################################################
+import openerp.http as http
+from openerp.http import request
+from os.path import join, abspath, exists
+import mimetypes
+
+
+class JsColor(http.Controller):
+    @http.route("/jscolor/<string:image>", type="http", auth="user")
+    def jscolor(self, image):
```

Comment:
this is needed because detectDir in https://github.com/OCA/web/pull/85/files#diff-8e64e0af10a444a4fcae715f63991980R39 fails, right? Wouldn't it be better then to patch this function to default to '/web_widget_color/static/lib/jscolor'?